src/styles/styles/styles.native.js

```diff
@@ -1,24 +1,24 @@
-import fontFamily from './fontFamily';
```

Comment:
I'm not sure about a change to this file. If some component requires different styles per platform, there should be a separate folder for these specific styles, not splitting global styles

Reply:
The `styles.native.js` file was just the base existing `styles.js` file I renamed it due to the folder and platform splitting structure there.
Not sure I understand `there should be a separate folder for these specific styles, not splitting global styles` please can you explain more @ArekChr

Reply:
If styles are different per specific platform, then a new file for those styles should be created instead of splitting global styles. e.g. styles/emojiText/styles.native.js.
@parasharrajat, could you correct me if I'm wrong?